board/safety.h

```diff
@@ -85,7 +85,7 @@ int safety_fwd_hook(int bus_num, CANPacket_t *to_fwd) {
 }
 
 bool get_longitudinal_allowed(void) {
-  return controls_allowed && !gas_pressed_prev;
+  return controls_allowed && !gas_pressed_prev && !brake_pressed_prev;
```

Comment:
Will need to allow braking on cars that are gas/brake controlled and not acceleration controlled.

On both GM and certain Honda, once you release the brake the car will lurch forward as openpilot's brake request ramps up.